cowamm/profitability/lp_prices/cow_amm_lp_4420687.sql

```diff
@@ -0,0 +1,105 @@
+-- Issues with the other version (4420687)
+-- Trying by accessing only once the logs table to get assets reserve
+-- However for periods longer than 1 month, the query exceeds cluster capacity
+-- The issue is with the join between both reserves_delta tables
+-- Although this table is only 4000 rows at most (~10kb)
+
+--Parameters
+--  {{blockchain}}: The blockchain to query
+--  {{start}}: The start date of the analysis
+--  {{end}}: The end date of the analysis. date(Timestamp) <= date(timestamp '{{end}}').
+--      For a 1 day period, {{end}} = {{start}}
+
+with date_range as (
+    select t.day
+    from
+        unnest(sequence(
+            date(timestamp '{{start}}'),
+            date(timestamp '{{end}}')
+        )) t (day) --noqa: AL01
+),
+
+--use the materialized view of query_3959044
+cow_amm_pool as (
+    select
+        created_at,
+        token_1_address as token0,
+        token_2_address as token1,
+        address as contract_address
+    from query_3959044
```

Comment:
Make it explicit that the materialized version needs to be used.

```suggestion
    from dune.cowprotocol.result_balancer_co_w_am_ms
```